0.5.0 fails to compile

I believe something happened around the `syn` crate, because the build fails with this reason:
```
   Updating crates.io index
   Compiling proc-macro2 v1.0.24
   Compiling unicode-xid v0.2.1
   Compiling syn v1.0.58
   Compiling quote v1.0.8
   Compiling delegate v0.5.0 (/home/mzs/.cargo/registry/src/github.com-1ecc6299db9ec823/delegate-0.5.0)
error[E0432]: unresolved import `syn::export`
  --> src/lib.rs:93:10
   |
93 | use syn::export::TokenStream2;
   |          ^^^^^^ could not find `export` in `syn`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: could not compile `delegate`

To learn more, run the command again with --verbose.
```

Also, could you enhance your CI with some periodic compilation?

Thanks a lot!

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -87,10 +87,10 @@
 extern crate proc_macro;
 
 use proc_macro::TokenStream;
+use proc_macro2::TokenStream as TokenStream2;
 
 use quote::quote;
 use std::collections::HashMap;
-use syn::export::TokenStream2;
 use syn::parse::ParseStream;
 use syn::spanned::Spanned;
 use syn::Error;

